charts/seed-operatingsystemconfig/original/values.yaml

```diff
@@ -18,8 +21,10 @@ kubernetes:
   version: 1.11.2
 worker:
   name: cpu-worker
-# cri:
+  cri:
 #   name: containerd
+#   labels:
+#   - worker.gardener.cloud/containerruntime/gvisor=true
```

Comment:
Why does the node label not look like this : "worker.gardener.cloud/containerruntime=gvisor".  This way we would not have to introduce a new label for each runtime. 
[RuntimeClass.scheduling.NodeSelector](https://github.com/kubernetes/kubernetes/blob/f0323a2030c7adae0e0965a7d3b455dd416472a0/staging/src/k8s.io/api/node/v1beta1/types.go#L84)   should be working as well with the aforementioned label structure. What was the rational behind structuring the label like that?
The label was explicitly mentioned in the [GEP](https://github.com/gardener/gardener/blob/master/docs/proposals/10-shoot-additional-container-runtimes.md) - so I am wondering if I miss something here.

Reply:
I think the reason is that there can be multiple container runtimes and not only one.

Reply:
Thanks Rafael, makes sense (as both these labels get added to the node eventually)